src/components/Label.tsx

```diff
@@ -2,18 +2,25 @@
 import React from 'react';
```

Comment:
Label is general component that shouldn't be touched. Create new component like TranscationLabelStatus or whatever that will use Label component under the hood